webservices/common/models/costs.py

```diff
@@ -58,9 +63,9 @@ class Electioneering(db.Model):
     disbursement_amount = db.Column('reported_disb_amt', db.Numeric(30, 2), index=True)
     purpose_description = db.Column('disb_desc', db.String)
     report_year = db.Column('rpt_yr', db.Integer, index=True)
-
-    committee = utils.related_committee('committee_id')
-    candidate = utils.related_candidate('candidate_id')
+    election_type = db.Column('election_tp', db.String)
```

Comment:
I was expecting P, G, S and R or something like that for election_type, instead there is:
```
 election_tp 
─────────────
 P2008
 P2016
 S
 O2011
 C
 G2012
 P2010
 P2014
 R
 G2008
 P2012
 R2008
 O2008
 G2014
 O2010
 G
 P
 G2010
```
I am not sure why sometimes it includes the year and sometimes it doesn't